src/renderer/views/Watch/Watch.js

```diff
@@ -1233,6 +1246,12 @@ export default defineComponent({
         return
       }
 
+      if (this.blockVideoAutoplay) {
+        showToast(this.$t('Canceled next video autoplay due to inactivity'))
```

Comment:
This should stay visible for longer (1 hour?)
New users (new to this setting) will take time to realize the playlist is stopped and switch back and wondering why it's stopped
Also the text should mention the setting name if possible, but at least should include the value (e.g. 3h)

Reply:
How about this?

![Screenshot_20241204_010228](https://github.com/user-attachments/assets/8353ed2a-e6ce-4fa2-874d-f3a95fa46e1a)

Reply:
Looks better